Assets/MixedRealityToolkit/_Core/Definitions/Devices/MixedRealityInteractionMapping.cs

```diff
@@ -0,0 +1,120 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Internal.Definitions.Utilities;
+using Microsoft.MixedReality.Toolkit.Internal.Interfaces;
+using Microsoft.MixedReality.Toolkit.Internal.Interfaces.InputSystem;
+using System;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Internal.Definitions.Devices
+{
+    /// <summary>
+    /// Maps the capabilities of controllers, linking the Physical inputs of a controller to a Logical construct in a runtime project<para/>
+    /// <remarks>One definition should exist for each physical device input, such as buttons, triggers, joysticks, dpads, and more.</remarks>
+    /// </summary>
+    [Serializable]
+    public class MixedRealityInteractionMapping<TReadingType> : IMixedRealityInteractionMapping
```

Comment:
I was going to comment on changes to this file but the meta must have gotten destroyed/corrupted.  Is there a way to fix so the diff is correct?